packages/fluentui/react-northstar/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -126,7 +134,7 @@ const AccordionTitle: React.FC<WithAsProp<AccordionTitleProps>> &
     mapPropsToBehavior: () => ({
       hasContent: !!content,
       canBeCollapsed,
-      as,
+      as: String(as),
```

Comment:
https://github.com/microsoft/fluentui/blob/6525619bf5e6c844406ed3cb9ad3103237eff89f/packages/fluentui/accessibility/src/behaviors/Accordion/accordionTitleBehavior.ts#L48

String is expected in the behavior